opencti-platform/opencti-front/lang/back/fr.json

```diff
@@ -204,7 +204,14 @@
   "Display name": "Nom d'affichage",
   "DLL service": "Service DLL",
   "Do you want to delete this support package?": "Voulez-vous supprimer ce paquet de support ?",
+  "Draft change": "Changement en brouillon",
   "Draft ids": "Identifiants de draft",
+  "Draft name": "Nom du projet",
```

Comment:
french translations don't seem right, should be "brouillon" I think and not "projet"